recipes/tests.rb

```diff
@@ -99,6 +99,13 @@
   end
 end
 
+if node['cfncluster']['os'] == 'alinux' || node['cfncluster']['os'] == 'centos7'
+  execute 'check efa_install' do
+    command "ls /opt/amazon/efa || exit 1"
+    user node['cfncluster']['cfn_cluster_user']
+  end
+end
+
```

Comment:
What about the check on the driver (the one instance type dependent)?

Reply:
As we discussed last week, the driver needs to be enabled in the LaunchTemplate which we can't change for the kitchen tests. This'll be tested in the integration test.